theme/devtools/devserver-graphql-simulator.js

```diff
@@ -0,0 +1,36 @@
+const { makeExecutableSchema, addMockFunctionsToSchema } = require('graphql-tools');
```

Comment:
Doesn't seem like this file is getting picked up by `prettier` or `eslint`. Can we check why that is?

Reply:
@DrewML I'll check. But as I noted, files in `devtools/` are marked for deletion anyway.